opentracing_instrumentation/client_hooks/celery.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import absolute_import
+
+from distutils.version import LooseVersion
```

Comment:
How common is it to depend on distutils like this? Could it create problems?  Maybe this import could be done in try/catch block  below?

Reply:
> How common is it to depend on distutils like this?

There are 500k+ [matches on GitHub](https://github.com/search?l=Python&q=from+distutils.version+import+LooseVersion&type=Code), so I assume it can be considered as common.

> Could it create problems? Maybe this import could be done in try/catch block below?

No, it couldn't, at least now, since `distutils` package is a part of [The Python Standard Library](https://docs.python.org/3/library/distutils.html).

Also, I checked [PEP 594 -- Removing dead batteries from the standard library](https://www.python.org/dev/peps/pep-0594/).
`distutils` was not deprecated, but I found the following text in the [discussions section](https://www.python.org/dev/peps/pep-0594/#discussions):
> At one point, distutils was mentioned in the same sentence as this PEP. To avoid lengthy discussion and delay of the PEP, I decided against dealing with distutils. Deprecation of the distutils package will be handled by another PEP.

I can replace using of `distutils` with some `split`-based logic or something else, but I think that it's reasonable to keep current implementation until `distutils` is marked as deprecated explicitly.

Reply:
Couldn't this logic be based on capabilities instead of explicit version checking (a bad practice)?

Reply:
It's done.